frontend/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -21,6 +21,7 @@ export const RadioGroup = <T extends string | number>({
 					borderColor: 'var(--color-purple)',
 					backgroundColor: 'var(--color-purple)',
 					color: 'var(--text-primary-inverted)',
+					...style,
```

Comment:
I wonder if we should have different props for the styles applied to the labels vs the wrapper on line 49?

Seems like it can cause some confusion if we wanted to apply the styles to one but not the other. It doesn't look like we are using the `style` on line 49 in other instances so maybe we can remove it? So something like:
```
<div className={styles.radioGroupWrapper}>
    {labelDivs}
</div>
```

Reply:
```html
<div style={wrapperStyle} className={styles.radioGroupWrapper}>
  {labelDivs}
</div>
`

It turns out the style on line 49 had an effect. I ended up introducing `labelStyle` and wrapperStyle props. That should clarify things